tests/SshTest.php

```diff
@@ -164,3 +164,10 @@
 
     assertMatchesSnapshot($command);
 });
+
+it('can login with a password', function () {
```

Comment:
could you also provide a fail scenario with an invalid password?

Reply:
sorry i'm not at home right now i couldn't test it right now